amundsen_application/static/js/components/TableDetail/ColumnListItem/index.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
```

Comment:
This component is missing the support for notifications that existed in `DetailListItem`, I think it goes in that "actions" section?

Reply:
This particular branch hasn't been merged with notifications/search_v2 yet. I was going to add those things in after merge. Will fix variable reference as well.

Reply:
I added the `Request Column Description` dropdown, but it looks a little different from mocks. Can touch up here, or change on master first and then merge down. The former being less work.

Reply:
I don't mind either option.